modules/core/client/components/Map/constants.js

```diff
@@ -0,0 +1,12 @@
+import osmStyle from './osm.json';
+
+export const getMapboxToken = () => window.settings?.mapbox?.publicKey;
```

Comment:
`const MAPBOX_TOKEN = window.settings?.mapbox?.publicKey;` gets back as undefined when imported. 🤷

I don't want to include in bundles either via Webpack. Not sure if this file is best for `getMapboxToken()` tho or should I move it to `utils.js`?

Reply:
I think its fine for now. There are some other ideas floating around for a more well defined settings structure.